.github/workflows/automerge.yaml

```diff
@@ -0,0 +1,46 @@
+# Via: https://github.com/pascalgn/automerge-action
+
+name: Automerge
+
+# Enable while working on this file
+on:
+    push:
+        paths: 
+            - "automerge.yaml"
+
+# Enable to automerge the daily data updates
+# on:
+#     pull_request:
+#         types:
+#             - labeled
+#     check_suite:
+#         types:
+#             - completed
+
+env:
+    GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+
+jobs:
```

Comment:
Should this have a conditional that verifies a few things?

- The PR’s branch name and/or authorship matches the ZoneDBot pattern defined in the other workflow?
- The pull request’s checks have all passed?

Reply:
This is currently commented out, did you see it?

```
on:
     pull_request:
         types:
             - labeled
     check_suite:
         types:
             - completed
```

My plan was to uncomment it prior to merging the PR.

To my knowledge, this 1) Ensures that the PR has a label (and `MERGE_LABELS: "automerge"` ensures it's looking for the `automerge` label), and 2) Only gets triggered if the tests pass (check_suite = completed).

I added this as well: `if: github.actor == 'ZoneDBot'`

Reply:
Nice!